flake/modules.nix

```
@@ -24,31 +23,29 @@
       };
 
     homeModules.stylix =
+      # deadnix: skip
       { pkgs, ... }@args:
```

again, this deadnix skip is unnecessary 
```suggestion
      args:
```

---

oh right, @MattSturgeon is there a better way to get around this?